app/src/main/java/com/rockthevote/grommet/ui/registration/UnknownDataSource.kt

```diff
@@ -0,0 +1,19 @@
+package com.rockthevote.grommet.ui.registration
+
+import com.rockthevote.grommet.data.db.model.GeoLocation
+
+/**
+ * TODO: I don't know where this data is derived from?
+ *
+ * DO NOT MERGE THIS, it is a placeholder while I figure out where data comes from
+ * for the API
+ */
+data class UnknownDataSource(
+    val partnerOptInEmail: Boolean,
```

Comment:
partnerOptInEmail, partnerOptInSms and partnerOptInVolunteer all just use default values , pointed out the locations in earlier comments but basically [here](partner_opt_in_email) and [here](PARTNER_OPT_IN_SMS)